mlir/include/mlir/Interfaces/DataLayoutInterfaces.h

```diff
@@ -84,6 +90,24 @@ Attribute getDefaultGlobalMemorySpace(DataLayoutEntryInterface entry);
 /// DataLayoutInterface if specified, otherwise returns the default.
 uint64_t getDefaultStackAlignment(DataLayoutEntryInterface entry);
 
+/// return max vector op width from the specified DataLayoutEntry. If the
```

Comment:
Nit: please capitalize the first word in the sentence.

Reply:
Fixed it.